preferences/linux/metadata/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: instancetype.kubevirt.io/v1beta1
+kind: VirtualMachinePreference
+metadata:
+  name: metadata
+  annotations:
+    tags: "hidden,kubevirt,linux"
+    iconClass: "icon-linux"
+    openshift.io/display-name: "Linux BIOS Guest"
```

Comment:
Can you drop `BIOS` here, while it's enabled by default I don't know if we need to call it out in the display name anymore.

Reply:
Done.